packages/client/src/subscriptions/deleteSubscription.ts

```diff
@@ -0,0 +1,26 @@
+import client, { adaptError } from '../helpers/client';
+import join from 'proper-url-join';
+import type { DeleteSubscription } from './types';
+
+/**
+ * Method responsible for sending a delete all subscriptions request to MKT API containing the subscription id and email hash of the user to be unsubscribed.
+ *
+ * @function deleteSubscription
+ * @memberof module:subscriptions/client
+ *
+ * @param query - Query object.
+ * @param config - Custom configurations to send to the client instance (axios).
+ *
+ * @returns Promise object.
+ */
+
+const deleteSubscription: DeleteSubscription = (query, config = undefined) => {
```

Comment:
It is unnecessary to say `config = undefined` because if no value for config is passed, it will undefined.

Reply:
Source code updated with suggestion.